addons/mail/static/src/new/discuss/sidebar.js

```diff
@@ -115,4 +116,14 @@ export class Sidebar extends Component {
             });
         });
     }
+
+    filteredThreads(category) {
+        if (!this.state.quickSearchVal) {
+            return category.threads;
+        }
+        return category.threads.filter((threadLocalId) => {
+            const thread = this.messaging.state.threads[threadLocalId];
+            return thread?.name.includes(this.state.quickSearchVal);
```

Comment:
thread can be undefined here?

Reply:
It looks like it should not be possible if state and relations are properly managed

Reply:
`thread.name` however can be empty for sure, when using group chat and name comes from name of members (should use `displayName`).

Reply:
name may not exist

Reply:
but then the `?` is misplaced

Reply:
In the end just removed it. Threads always have a name (old tests are being adapted, thanks to props validation of autoresize input)